Http.cpp

```diff
@@ -194,6 +196,47 @@ unsigned int HTTP::readVoltage(){
   return atoi(voltage);
 }
 
+void HTTP::batteryState(char *voltage){
```

Comment:
What about renaming this method to `batteryPercentage`?

Reply:
Sure that would make sense.